--- gmpc-coveramazon-0.14.0/src/Makefile.am.wig	2007-02-03 18:24:45.000000000 +0100
+++ gmpc-coveramazon-0.14.0/src/Makefile.am	2007-02-19 22:18:57.000000000 +0100
@@ -1,7 +1,6 @@
-#bin_PROGRAMS=osdplugin_la
-libdir=${prefix}/share/gmpc/plugins/
+pluginsdir=$(libdir)/gmpc/
 
-lib_LTLIBRARIES = caaplugin.la
+plugins_LTLIBRARIES = caaplugin.la
 
 caaplugin_la_SOURCES=	\
 	plugin.c 
@@ -10,9 +9,3 @@
 caaplugin_la_LIBADD= @gmpccaa_LIBS@
 caaplugin_la_CFLAGS= @gmpccaa_CFLAGS@  
 caaplugin_la_LDFLAGS= -module -avoid-version 
-
-install: 
-	@$(MAKE)
-	mkdir -p ${DESTDIR}${libdir}
-	if test -f .libs/caaplugin.so; then cp .libs/caaplugin.so ${DESTDIR}${libdir}; fi
-	if test -f .libs/caaplugin.dll; then cp .libs/caaplugin.dll ${DESTDIR}${libdir}; fi
